Client/core/CEntryHistory.h

```diff
@@ -66,7 +66,7 @@ class CEntryHistory
     // Return a specific entry from history
     CEntryHistoryItem* Get(unsigned int index)
     {
-        auto& iter = std::next(m_entries.begin(), index);
+        auto iter = std::next(m_entries.begin(), index);
```

Comment:
Unnecessary change. Leave it as a reference

Reply:
Can I close this?

Reply:
Yes, of course